build.sbt

```diff
@@ -1,22 +1,17 @@
 import sbtrelease.Version
 
 val embeddedKafkaVersion = "2.2.0"
```

Comment:
don't we also want force kafka version do be `2.0.1` Currently the one on the classpath is `2.2.0`

Reply:
Getting exceptions if downgrade this to 2.0.0 ```unresolved dependency: io.github.embeddedkafka#embedded-kafka-streams_2.12;2.0.0: not found```

Reply:
I mean only downgrade kafka not the embedded lib. Its expected from the client to specify the concrete kafka version?